### PR TITLE
Add theorem setis

### PR DESCRIPTION
**Pull request by emmett-w**
This commit also deletes two extra blank lines.

Theorem setis proves that induction works on a recursively generated set B, which could be, for example, On or Pg.

### CODE REVIEW

**Review by tirix — APPROVED** [review]
Looks good!

**Review by avekens — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -735845,14 +735844,28 @@ the relevant induction theorem (such as ~ tfi ) to the other class.
...
+  ${
+    $d F a b $.  $d a ps $.  $d b ch $.  $d b A $.
+    setis.1 $e |- B = setrecs ( F ) $.
+    setis.2 $e |- ( b = A -> ( ps <-> ch ) ) $.
+    setis.3 $e |- ( ph -> A. a ( A. b e. a ps -> A. b e. ( F ` a ) ps ) ) $.
+    $( Version of ~ setrec2 expressed as an induction schema.  This theorem is
+       a generalization of ~ tfis3 .  (Contributed by Emmett Weisz,
+       27-Feb-2022.) $)
+    setis $p |- ( ph -> ( A e. B -> ch ) )
+$=
```
Comment (benjub) [inline set.mm:735856]: can you put $= on the line above ?

Comment (emmett-w) [reply set.mm:735856]: Sure, I just did it.

**Review by benjub — APPROVED** [review]

**Comment by emmett-w** [comment]
@benjub Thank you!

**Pull request merged**